src/MongoDB.EntityFrameworkCore/Extensions/MongoPropertyExtensions.cs

```diff
@@ -155,6 +145,16 @@ public static void SetBsonRepresentation(
         => property.FindAnnotation(MongoAnnotationNames.BsonRepresentation)?.GetConfigurationSource();
 
 
+    /// <summary>
+    /// Gets the <see cref="DateTimeKind"/> of the <see cref="DateTime"/> property is mapped to when targeting MongoDB.
+    /// </summary>
+    /// <param name="property">The <see cref="IReadOnlyProperty"/> to obtain the <see cref="DateTimeKind"/> for.</param>
+    public static DateTimeKind GetDateTimeKind(this IReadOnlyProperty property)
```

Comment:
We should chat about this. :-) (But not as part of this PR.)

Reply:
This was existing code that was just moved in the file but yeah we can talk about that offline.